src/betterproto/__init__.py

```diff
@@ -889,6 +889,14 @@ def __deepcopy__(self: T, _: Any = {}) -> T:
                 kwargs[name] = deepcopy(value)
         return self.__class__(**kwargs)  # type: ignore
 
+    def __copy__(self: T, _: Any = {}) -> T:
+        kwargs = {}
+        for name in self._betterproto.sorted_field_names:
+            value = self.__raw_get(name)
+            if value is not PLACEHOLDER:
+                kwargs[name] = value
+        return self.__class__(**kwargs)  # type: ignore
```

Comment:
Can you add some copy tests?

Reply:
there appears to be one already, I copied it and added pickling as well:

```python
﻿def test_pickle_copyability():
    msg = PickleMessage(bar=12, baz=["hello"])
    deserialized = pickle.loads(pickle.dumps(msg))

    copied = copy(msg)
    assert msg == copied == deserialized
    assert msg is not copied is not deserialized
    assert msg.baz is copied.baz #is deserialized.baz

    deepcopied = deepcopy(msg)
    assert msg == deepcopied == deserialized
    assert msg is not deepcopied is not deserialized
    assert msg.baz is not deepcopied.baz is not deserialized.baz
```
However the commented assert seems to evaluate false... I think that's correct?? Since pickling and unpickling would technically mean the field is now a different object

Reply:
Can the lines in https://github.com/danielgtaylor/python-betterproto/blob/6b36b9ba9ffda8a7b3b488b774729c834db8ae0d/tests/test_features.py#L702-L718 be moved with the new tests?

Reply:
Done

I also moved across the recursive message tests as they now are pickled before testing as well